README.md

```diff
@@ -53,6 +53,34 @@ describe("BananService") {
 
 Each `it` block should have only one assertion. This is in line with the _Isolated_ principle.
 
+```swift
+describe("Movie") {
+  context("when initializing from dictionary") {
+  
+    let name = "The Dark Knight Rises"
+    let genre = "Action"
+    let year = 2012
+    let movie: Movie!
+    
+    beforeEach {
+      movie = Movie(dict: ["name": name,
+                           "genre": genre,
+                           "year": year])
```

Comment:
This is not consistent with the indentation defined in [our Swift styleguide](https://github.com/iflix-letsplay/swift-style-guide).

Having said that, I think that in the context of this example, to keep the code compact we could inline the whole definition:

```swift
movie = Movie(dict: ["name": name, ...])
```